docs/devices/ZB-IR01.md

```diff
@@ -0,0 +1,1080 @@
+---
+title: "Easyiot ZB-IR01 control via MQTT"
+description: "Integrate your ZB-IR01 via Zigbee2MQTT with whatever smart home infrastructure you are using without the vendor's bridge or gateway."
+addedAt: 2023-11-23T16:23:11Z
+pageClass: device-page
+---
+
+<!-- !!!! -->
+<!-- ATTENTION: This file is auto-generated through docgen! -->
+<!-- You can only edit the "Notes"-Section between the two comment lines "Notes BEGIN" and "Notes END". -->
+<!-- Do not use h1 or h2 heading within "## Notes"-Section. -->
+<!-- !!!! -->
+
+# Easyiot
+
+|     |     |
+|-----|-----|
+| Model | ZB-IR01 |
+| Vendor  | [Easyiot](/supported-devices/#v=Easyiot)  |
+| Description | This is an infrared remote control equipped with a local code library, supporting devices such as air conditioners, televisions, projectors, and more. |
+| Exposes | send command, recv command, linkquality |
+| Picture | ![ZB-IR01](https://www.zigbee2mqtt.io/images/devices/ZB-IR01.jpg) |
+
+
+<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
+## Notes
+## 1 Protocol Overview
+
+### 1.1 Protocol Description
+
+This agreement document is applicable to the following two methods:
```

Comment:
If I understood correctly, this is also written in a document/pdf somewhere? If yes, I prefer that you link to it from here.